c2corg_ui/static/js/editing/documentediting.js

```diff
@@ -627,7 +627,9 @@ app.DocumentEditingController.prototype.confirmSave = function(isValid) {
 
   const data = this.scope[this.modelName];
   const doctype = app.utils.getDoctype(data['type']);
-  if ((doctype != 'routes') && (doctype != 'waypoints') && (doctype != 'books') && (doctype != 'xreports') && (doctype != 'users') && (doctype != 'areas')) {
+  if (doctype == 'users') {
```

Comment:
Why not use the default `presetQuality` method also for users?
This returns a score of 1, which is not medium quality, but anyway, why not return a score of 2?

Reply:
There is no 'quality' field for users.
Normally the default value (1) should never be set since all other type of documents have their own computation function. We can set it to 2 if you want.